news/3493.bug

```diff
@@ -0,0 +1 @@
+Reset update.date_testing when editing builds.
```

Comment:
https://bodhi.fedoraproject.org/docs/developer/index.html#release-notes
"It should not end with a `.` (full stop)"